app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
       log_in user
       redirect_to root_url
     else
-      flash[:danger] = t ".danger"
+      flash.now[:danger] = t ".danger"
```

Comment:
biết sự khác nhau của now và không now không đó.

Reply:
.now thì nó báo lỗi ở trang đấy thôi ạ , không now thì khi redirect trang khác r nó vẫn hiển thị thông báo ạ

Reply:
why are you use "now" here :v

Reply:
@hungct-1121  dạ khi login báo lỗi thì chỉ báo trên trang login thôi ạ , chứ k để now ra home nó vẫn còn ạ